glide.yaml

```diff
@@ -52,3 +52,8 @@ import:
   version: master
 - package: github.com/pborman/uuid
   version: ^1.1.0
+- package: honnef.co/go/tools/cmd/staticcheck
+  version: 8ed405e85c65fb38745a8eafe01ee9590523f172
```

Comment:
Add an issue that describes why you did this. It can be annoying to figure out why deps are pinned at particular hashes without it.

Reply:
good advice

Reply:
https://github.com/uber/zanzibar/issues/398